AmplifyPlugins/Logging/Tests/AWSCloudWatchLoggingPluginTests/RotatingLoggerTests.swift

```diff
@@ -123,6 +123,22 @@ final class RotatingLoggerTests: XCTestCase {
         try assertSingleEntryWith(level: level, message: message)
     }
     
+    /// Given: a rotating logger
+    /// When: get log batch is called
+    /// Then: a list of log batches is returned
+    func testRotatingLogReturnsLogBatches() async throws {
```

Comment:
Similar to previous test, i think we can add here in the precondition that
initially 1 log exists, then after 1 more log entry, the total number of log entries should increase by 1